src/components/overlays/InviteForm/index.tsx

```diff
@@ -130,7 +130,11 @@ export const InviteForm = ({
           <Button variant="outlined" onClick={(e) => handleOverlayClose(e)}>
             {`${t('global.actions.cancel')}`}
           </Button>
-          <Button name="send" disabled={inpValid[4]} onClick={doSubmit}>
+          <Button
+            name="send"
+            disabled={inpValid[4] || state === 'busy'}
+            onClick={doSubmit}
+          >
```

Comment:
@nidhigarg-bmw  & @manojava-gk  thats something you both could please discuss jointly - it is something which should in general be a standard implementation of our action buttons connected to an backend API. @nidhigarg-bmw please review the pr from Mano.

Reply:
@jjeroch Me and @manojava-gk  collectively vote for Loading button. In all places we are using currently Loading button and only here we are using Loader.